proposal/report_group_XX.tex

```diff
@@ -8,31 +8,109 @@
 \newcommand{\todo}[1]{\textcolor{red}{#1}}
 
 \title{CS482/682 Final Project Report Group \todo{XX}\\
-	\large \todo{your project title here}}
+  \large Using Unsupervsed Learned to provide a Competive Edge in a Classification Competition}
 
-\author{Your names/JHED here}
+\author{Tianyu Ding, Ivan Liao, Dylan Madisetti, Alex Sun}
 
 \date{}
 
 \begin{document}
-	
-	\maketitle
-	
-	% do not write an abstract
-	\section{Introduction}
-	\paragraph{Background} 
-	\paragraph{Related Work} \todo{e.g. previous supervised approaches, other unsupervised methods etc}
-	
-	\section{Methods}
-	\paragraph{Dataset} \todo{BRIEFLY describe the dataset and any pre-processing you may use that is special (e.g. downsample all images by x8 to enable colaboratory training etc.)}
-	
-	\paragraph{Setup, Training and Evaluation} \todo{IMPORTANT: you can change this paragraph to better fit your project. Questions to answer: What architecture did you chose and why? How did you accomplish un- or self-supervised learning? What tweaks were necessary to make this happen, e.g. custom layer designs, auxiliary tasks, etc.?}
-	
-	
-	
-	\section{Results}
-	
-	
-	\section{Discussion} \todo{make sure you use this section to showcase your understanding of the results you achieved and whether you could have / should have done things differently }
-
-\end{document}
\ No newline at end of file
+
+  \maketitle
+
+  % do not write an abstract
+  \section{Introduction}
+

```

Comment:
- first of all, we should have a brief problem statement, maybe we can paraphrase the description from the kaggle description?